L1Trigger/L1TMuonBarrel/src/L1MuBMSEU.cc

```diff
@@ -82,7 +82,7 @@ L1MuBMSEU::~L1MuBMSEU() {
 //
 // run SEU
 //
-void L1MuBMSEU::run(const edm::EventSetup& c) {
+void L1MuBMSEU::run(const L1TMuonBarrelParams& params) {
   if (L1MuBMTFConfig::Debug(3))
     cout << "Run SEU " << m_ext << " " << m_startTS_Id << endl;
```

Comment:
@Dr15Jones  You may take the chance to remove these cout?

Reply:
(They are steered by `L1MuBMTFConfig::Debug(3)`: as such they can probably stay, taking also into account that this is not the main topic of this PR...)